test/index.js

```diff
@@ -813,3 +816,9 @@ test('contramap', function() {
 
   eq(Z.contramap(length, inc)('abc'), 4);
 });
+
+test('errors', function() {
+  throws(function() { Z.chainRec(Maybe); },
+         TypeError,
+         'ChainRec.methods.chainRec(...) is not a function');
+});
```

Comment:
I'm not sure it's a good idea to make assertions about errors thrown unintentionally. The error message may differ from one implementation to the next.